games-misc/cowsay/Manifest

```diff
@@ -1 +1 @@
-DIST cowsay-3.04.tar.gz 31961 BLAKE2B 93e6e9cb31b8902ee6bccda0f997fa651fa83e0504ccab3e7fbbc5cfddc11256f6adea1ea1b4e2bcf8085746eb4194e8332bfee886d1f3d05ad2b6ac3422eea8 SHA512 99d21ef9a7c7c76056cb7acb2c771fbdabfedde4f6edb5d3035337c59f9e88944690d1b43282c32827ad79339199716366c8be73b0dea09063ec2d52220c1b51
```

Comment:
```
 ________________________________________ 
/ fwiw this version existed without      \
| issues for a long time, maybe keep it  |
| for a bit so people can revert if the  |
| new one broke their important workflow |
\ before they lose their job over it     /
 ---------------------------------------- 
        \   ^__^
         \  (00)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
```

Reply:
Yeah, I always try to avoid cleaning up old immediately in situations like this (and I’m trying to be more conservative about it in general even for ~arch bumps)

Reply:
```
 _________________________________ 
( but my version is superior1!!1! )
 --------------------------------- 
        o   ^__^
         o  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||

```
Of course that's a reasonable approach. Fixed locally.